api/app/routers/hfi_calc.py

```diff
@@ -361,6 +368,42 @@ async def get_fire_centres(response: Response):
         raise
 
 
+@router.get('/add-station/{fire_centre_id}', response_model=HFIAddStationOptionsResponse)
+async def get_add_station_options(fire_centre_id: int, response: Response, _=Depends(authentication_required)):
+    """ Returns lists of planning areas, stations and fuel types for adding a station. """
+
+    try:
+        logger.info('/hfi-calc/add-station')
+        # we can safely cache the options, as they don't change them very often.
+        response.headers["Cache-Control"] = "max-age=86400"
```

Comment:
I don't actually think we can cache this - if we need this endpoint, it's because station information is changing. (also - keep in mind when writing the post - it may be a good idea to invalidate the cache - since changing stations could mean that we have a stale station lits)